### DISCUSSION TITLE
Expose all express-rate-limit options

### DISCUSSION THREAD

**Discussion by geminigeek** [payloadcms/payload#3561] [Feature Requests & Ideas]
hi,

currently payload only have few settings allowed for express-rate-limit , if we can expose all options most important "store" so we can use external store like redis with "express-rate-limit" , current implementation  fail to provide protection in case of running 2 instance of application with pm2.

**Comment by DanRibbens** [comment]
Hey @geminigeek, 
What you're saying makes a lot of sense. We're about to go through our roadmap so I've labeled this for consideration for the team.

Thanks!
